napari/layers/_base_layer/model.py

```diff
@@ -92,6 +95,39 @@ def name(self, name):
         self._name = name
         self.events.name()
 
+    @property
+    def indices(self):
+        """Tuple of slice objects for slicing arrays on each dimension."""
+        return self._indices
+
+    @indices.setter
+    def indices(self, indices):
+        if indices == self.indices:
+            return
+        self._indices = indices[-self.ndim:]
+        self._set_view_slice()
+
+    @property
+    def cursor_position(self):
+        """tuple: 2-tuple of cursor in image coordinates. Setter does
+        transformation from canvas coordinates to image coordinates
```

Comment:
I suggest:

```
2-tuple of cursor coordinates in image (pixel) space.

The setter expects the coordinates in canvas (real) space and transforms them
to image space, which is how they are stored internally.
```

or vice versa, I'm not sure what's happening, but that sort of stuff. =)

Reply:
i'm eliminating this property as it was not very useful